src/store/servers.ts

```diff
@@ -13,28 +13,37 @@ export const servers: IServer[] = [
   {
     name     : 'Piplup (Main)',
     link     : 'https://piplup.smoo.it',
-    server   : { host: 'piplup.smoo.it', ip: '143.198.241.242' },
-    location : { flag: 'gb', name: 'United Kingdom' },
+    server   : { host: 'piplup.smoo.it' },
+    location : { flag: 'fi', name: 'Finland' },
     version  : linkTree('server', 'piplup/server', 'TeamPiplup/SmoOnlineServer'),
     settings : {
       Server        : { MaxPlayers: 10 },
-      Scenario      : { MergeEnabled: false },
+      Scenario      : { MergeEnabled: true },
       Shines        : { Enabled: true },
       PersistShines : { Enabled: true },
     },
   },
   {
     name     : 'Piplup (Capture Sync)',
     link     : 'https://piplup.smoo.it',
-    server   : { host: 'piplup.smoo.it', ip: '143.198.241.242', port: 1028 },
-    location : { flag: 'gb', name: 'United Kingdom' },
+    server   : { host: 'piplup.smoo.it', port: 1028 },
+    location : { flag: 'fi', name: 'Finland' },
     version  : linkTree('server', 'piplup/server', 'TeamPiplup/SmoOnlineServer'),
+  },
```

Comment:
The Capture Sync server has no settings defined here now, because they moved to the UDP server. It will show these settings when the JSON API is not working or the server is offline. At least the `MaxPlayers` setting should be defined here as a minimum.

Reply:
oh oops i missed that